model/versioning/Versioned.php

```diff
@@ -301,12 +308,13 @@ public function augmentSQL(SQLSelect $query, DataQuery $dataQuery = null) {
 		// Reading a specific stage (Stage or Live)
 		case 'stage':
 			$stage = $dataQuery->getQueryParam('Versioned.stage');
-			if($stage && ($stage != $this->defaultStage)) {
+			if($this->hasVersionedStage($stage) && $stage !== static::DRAFT_STAGE) {
```

Comment:
Some of the shop modules use 1 stage for versioning so they can always refer to the price of an item at the time of order.